cloud/aws/modules/ecs_fargate_service/main.tf

```diff
@@ -236,9 +236,15 @@ resource "aws_ecs_service" "service" {
     assign_public_ip = false
   }
 
+  # The circuit breaker will mark the deployment as failed when the service does
+  # not come up or is not marked as healthy after 10 tries. This prevents the
+  # deployment from pulling the civiform image over and over, which can cause
+  # DockerHub rate limiting. The 10 tries is the minimum value given that
+  # desired_count < 20 (desired_count is probably 1 for our use case).
```

Comment:
question (non-blocking): Just making sure I understand -- It's set to 10 because the desired task count is only 1?